sno/init.py

```diff
@@ -343,6 +343,9 @@ def init(
     write_repo_version_config(repo, repo_version)
     WorkingCopy.write_config(repo, wc_path, bare)
 
+    # Enable writing to reflogs
+    repo.config["core.logAllRefUpdates"] = "always"
```

Comment:
so this automatically persists/writes into the repo `.git/config`? #til

Reply:
Yes it writes into the config file.